components/kyma-metrics-collector/pkg/process/process_test.go

```diff
@@ -37,7 +38,7 @@ const (
 	bigTimeout = 10 * time.Second
 
 	// KEB related variables.
-	kebRuntimeResponseFilePath = "../testing/fixtures/runtimes_response.json"
```

Comment:
Is the `../testing/fixtures/runtimes_response.json` file still used in the tests?

Reply:
Yes, it is used in a different test.